fontforge/python.c

```diff
@@ -19221,20 +19218,20 @@ static void SetPythonModuleMetadata( PyObject *module ) {
     char isodate[32];
     PyObject* pyver;
     PyObject* pydate;
-    int dt = FONTFORGE_VERSIONDATE_RAW;
+    time_t dt = FONTFORGE_MODTIME_RAW;
+    struct tm* modtime = gmtime(&dt);
 
     /* Make __version__ string */
-    snprintf(ver, sizeof(ver), "%u.%u git:%s",
-	     FONTFORGE_LIBFF_VERSION_MAJOR,
-	     FONTFORGE_LIBFF_VERSION_MINOR,
+    snprintf(ver, sizeof(ver), "%s git:%s",
```

Comment:
I guess that this is okay since we encode the version number in the git tag as appropriate.